tests/python/pytorch/graphbolt/test_item_sampler.py

```diff
@@ -610,3 +614,121 @@ def test_ItemSetDict_node_pairs_negative_dsts(batch_size, shuffle, drop_last):
     assert torch.all(src_ids[:-1] <= src_ids[1:]) is not shuffle
     assert torch.all(dst_ids[:-1] <= dst_ids[1:]) is not shuffle
     assert torch.all(negs_ids[:-1] <= negs_ids[1:]) is not shuffle
+
+
+def distributed_item_sampler_subprocess(
+    proc_id,
+    nprocs,
+    item_set,
+    num_ids,
+    batch_size,
+    shuffle,
+    drop_last,
+    drop_uneven_inputs,
+):
+    # On Windows, the init method can only be file.
+    init_method = (
+        f"file:///{os.path.join(os.getcwd(), 'dis_tempfile')}"
+        if platform == "win32"
+        else "tcp://127.0.0.1:12345"
+    )
+    dist.init_process_group(
+        backend="gloo",  # Use Gloo backend for CPU multiprocessing
+        init_method=init_method,
+        world_size=nprocs,
+        rank=proc_id,
+    )
+
+    # Create a DistributedItemSampler.
+    item_sampler = gb.DistributedItemSampler(
+        item_set,
+        batch_size=batch_size,
+        shuffle=shuffle,
+        drop_last=drop_last,
+        drop_uneven_inputs=drop_uneven_inputs,
+    )
+    data_loader = gb.SingleProcessDataLoader(item_sampler)
```

Comment:
what about `MultiProcessDL`? cannot be used?

Reply:
Could be used but not perfect.